src/credentials/sharedCredentials.ts

```diff
@@ -234,6 +234,13 @@ async function loadCredentialsFile(credentialsUri?: vscode.Uri): Promise<ReturnT
     return parseIni(await SystemUtilities.readFile(credentialsUri), credentialsUri)
 }
 
+/**
+ * The name of a section in a credentials/config file
+ *
+ * The is the value of `{A}` in `[ {A} ]` or `[ {B} {A} ]`.
+ */
+export type ProfileName = string
```

Comment:
`SectionName` might be less confusing. `profile` sections need to be handled differently from `sso-session` sections. But both are named.